es6/assignment.js

```diff
@@ -1,96 +1,97 @@
-import { PlanOutOpRandom } from "./ops/random";
 import { shallowCopy, forEach, hasKey } from "./lib/utils";
 
-class Assignment {
-  constructor(experimentSalt, overrides) {
-    if (!overrides) {
-      overrides = {};
+export default function provideAssignement(Random) {
```

Comment:
It's because the named export is the default export, which you can import as whatever name you choose. The main advantage of naming a default export function is that it'll make stack traces a bit easier to follow when the code is compiled & minified.